components/sup/Cargo.toml

```diff
@@ -28,6 +28,7 @@ habitat_core = { path = "../core" }
 habitat_depot_client = { path = "../builder-depot-client" }
 habitat-eventsrv-client = { path = "../eventsrv-client" }
 habitat-launcher-client = { path = "../launcher-client" }
+habitat-launcher-protocol = { path = "../launcher-protocol" }
```

Comment:
Instead of taking a direct dep on `habitat-launcher-protocol` I would instead re-pub the const that you want inside the client. This crate is supposed to be "hidden" to the supervisor (I should probably document that 😉)

You can see an example [here](https://github.com/habitat-sh/habitat/blob/master/components/launcher-client/src/lib.rs#L23)